site/en/docs/extensions/mv3/match_patterns/index.md

```diff
@@ -3,159 +3,76 @@ layout: 'layouts/doc-post.njk'
 title: 'Match patterns'
 seoTitle: 'Chrome Extensions match patterns'
 date: 2012-09-18
-updated: 2023-07-28
+updated: 2023-08-31
 description: Understanding URL match patterns in Chrome extensions.
 ---
 
-## Overview {: #overview }
-
-Extensions use match patterns to specify a group of URLs. They are mainly used to declare [host permissions][host-permissions] along with other permissions to use certain Chrome APIs or when injecting [content scripts][content-scripts]. Extensions also use match patterns in:
-
-- [`"web_accessible_resources.matches"`][war]
-- [`"externally_connectable.matches"`][ext-connect]
-
-## What is a match pattern? {: #what }
-
-A match pattern is a URL that begins with a permitted scheme, separated by a `://`, followed by a host and path. It can contain wildcard (`*`) characters. Most match patterns are composed of three parts:
+A match pattern is a URL with the following structure, used to specify a group of URLs:
 
 ```text
 &lt;scheme&gt;://&lt;host&gt;/&lt;path&gt;
 ```
 
-Each part can use wildcards `*`. Below is a detailed description:
+**scheme**: Must be one of `'http'`, `'https'`, `'file'`, `'ftp'`, or `'urn'`. The wildcard `*` only matches `http` or `https`.
```

Comment:
Are we sure about `urn`?

Reply:
I'm trying to find out.

Reply:
If I understood Devlin correctly, this is the correct list of schemes. https://source.chromium.org/chromium/chromium/src/+/main:extensions/common/url_pattern.cc;l=31-52;drc=8c08dd32bf007ca09f57b63b6b812478b44ac2c4

Reply:
Updated. Does it still need the single quotes?

Reply:
I'd say no quotes. I insist on quotes for manifest keys because they're part of what actually needs to be in the manifest. Quotes aren't part of a URL scheme.

Reply:
Done.